CONTRIBUTING.md

```diff
@@ -185,3 +160,36 @@ package.
   * Do all the GitHub Actions and CircleCI tests pass? If not, are they allowed to fail?
   * If applicable, has an entry been added into the changelog?
   * Can you check out the pull request and repeat the examples and tests?
+
+Other Tips
+----------
+
+- Behind the scenes, we conduct a number of tests or checks with new pull requests.
+  This is a technique that is called continuous integration, and we use GitHub Actions
+  and CircleCI. To prevent the automated tests from running, you can add ``[ci skip]``
+  to your commit message. This is useful if your PR is a work in progress (WIP) and
+  you are not yet ready for the tests to run. For example:
+
+      $ git commit -m "WIP widget [ci skip]"
+
+  - If you already made the commit without including this string, you can edit
+    your existing commit message by running:
+
+        $ git commit --amend
+
+- Unfortunately, GitHub Actions ignores ``[ci skip]`` for a PR, so we recommend
```

Comment:
I think only circle CI and readthedocs now ignore `skip ci`?

Reply:
It appears so.  I'll fix that.

Reply:
Ok, I went ahead and removed that bullet point since it's no longer really pertinent.